studio-docs/source/org/members.mdx

```diff
@@ -32,12 +32,13 @@ Only admins can assign roles to other members. You can see which role you have i
 | Invite members                                         | ✓ |   |   |   |   |
 | Remove members                                         | ✓ | ✓ |   |   |   |
 | View and edit billing information	                     | ✓ | ✓ |   |   |   |
-| Manage organization configuration (name, avatar, etc.)  | ✓ | ✓ |   |   |   |
+| Manage organization configuration (name, avatar, etc.) | ✓ | ✓ |   |   |   |
 | Delete         graphs                                  | ✓ |   |   |   |   |
 | Create, rename graphs                                  | ✓ |   | ✓ |   |   |
 | Manage graph integrations (Datadog, Slack, etc.)       | ✓ |   | ✓ |   |   |
 | View and manage graph API keys                         | ✓ |   | ✓ |   |   |
-| Modify schema checks configuration                      | ✓ |   | ✓ |   |   |
+| Modify schema checks configuration                     | ✓ |   | ✓ |   |   |
+| Push a new schema to your graph                        | ✓ |   | ✓ |   |   |
```

Comment:
what does "new" mean here? this makes it feel different from a new version of schema...

Reply:
is "push schemas to your graph" better?